MaterialLib/Fluid/Viscosity/WaterViscosityIAPWS.cpp

```diff
@@ -0,0 +1,172 @@
+/**
+ *  \copyright
+ *   Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *              Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * \file   WaterViscosityIAPWS.cpp
+ *
+ * Created on December 1, 2016, 1:41 PM
+ */
+
+#include "WaterViscosityIAPWS.h"
+
+#include <cmath>
+
+namespace MaterialLib
+{
+namespace Fluid
+{
+const double WaterViscosityIAPWS::_hi[4] = {1.67752, 2.20462, 0.6366564,
+                                            -0.241605};
+const double WaterViscosityIAPWS::_hij[6][7] = {
```

Comment:
It seems the both arrays are only needed in the cpp file. If so, there is no need to add them to the class definition, just make them static in the cpp.